src/styles/components/_home-search-section.scss

```diff
@@ -0,0 +1,13 @@
+@import '../abstracts/_abstracts-dir';
+
+.home-search {
```

Comment:
Name file must always match class name. But IMO it would be better to rename it to `home-search-tab`